### PR TITLE
feat: add contact form

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

**Review by lukashass — APPROVED** [review]
I guess this is only a good UX when the mailto handler is configured, otherwise you write something to then notice it doesn't work.

Since we can ignore desktop it might be ok though.

**Comment by anbraten** [comment]
Sure in the end you need to have a browser supporting the `mailto` link type (seems to be the case for all major ones). Desktop works for me as well?!

File: app/src/locales/de.json [hunk 1]
```diff
@@ -52,5 +51,8 @@
   "light": "Hell",
   "dark": "Dunkel",
   "next_stop": "Nächster Stop:",
-  "ok": "Ok"
+  "ok": "Ok",
+  "contact_title": "Möchtest du Feedback geben, eine Idee vorschlagen, ein Problem melden oder einfach nur Danke sagen? Wir freuen uns über deine E-Mail!",
```
Comment (janelucia) [inline app/src/locales/de.json:55]: Minor thing, but I think this should read "Nachricht" instead of E-Mail.

File: app/src/locales/de.json [hunk 2]
```diff
@@ -52,5 +51,8 @@
...
+  "send_email": "E-Mail senden",
```
Comment (janelucia) [inline app/src/locales/de.json:56]: here as well

File: app/src/locales/en.json [hunk 3]
```diff
@@ -52,5 +51,9 @@
   "light": "Light",
   "dark": "Dark",
   "next_stop": "Next stop:",
-  "ok": "Ok"
+  "ok": "Ok",
+  "contact_title": "Would you like to share feedback, propose an idea, report an issue, or simply express gratitude? Your email would be greatly appreciated!",
```
Comment (janelucia) [inline app/src/locales/en.json:55]: message instead of email

File: app/src/locales/en.json [hunk 4]
```diff
@@ -52,5 +51,9 @@
...
+  "contact_email_body": "Hi Kiel-Live Team 👋,\n\n",
+  "send_email": "Send email",
```
Comment (janelucia) [inline app/src/locales/en.json:57]: same

**Pull request merged**